### DISCUSSION TITLE
Δήλωση πρόθεσης της προαιρετικής παρουσίασης ΟΣΣ1

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#161] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ερευνητικού άρθρου στέλνοντας το τελικό αρχείο παρουσίασης

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη ΟΣΣ

**Comment by VagNta** [comment]
[Παρουσίαση_SDY60_VagNta.pptx](https://github.com/user-attachments/files/17377597/_SDY60_VagNta.pptx)

**Comment by VasilisKokkalis** [comment]
Δήλωση πρόθεσης προαιρετικής παρουσίασης.

[SENSING FINE – GRAINED HAND ACTIVITY WITH SMARTWATCHES.pptx](https://github.com/user-attachments/files/17450985/SENSING.FINE.GRAINED.HAND.ACTIVITY.WITH.SMARTWATCHES.pptx)

Laput, G., & Harrison, C. (2019, May). Sensing fine-grained hand activity with smartwatches. In Proceedings of the 2019 CHI Conference on Human Factors in Computing Systems (pp. 1-13).

**Comment by apapost** [comment]
Δήλωση πρόθεσης προαιρετικής παρουσίασης


[Uploading Αλληλεπίδραση Ανθρώπου -Ρομπότ.pptx…]()

**Comment by VagNta** [comment]
https://dl.acm.org/doi/abs/10.1145/3334480.3382923

My Body, My Avatar: How People Perceive Their Avatars in Social Virtual Reality
Authors: [Guo Freeman](https://dl.acm.org/doi/10.1145/3334480.3382923#), [Samaneh Zamanifard](https://dl.acm.org/doi/10.1145/3334480.3382923#), [Divine Maloney](https://dl.acm.org/doi/10.1145/3334480.3382923#), [Alexandra Adkins](https://dl.acm.org/doi/10.1145/3334480.3382923#)[Authors Info & Claims](https://dl.acm.org/doi/10.1145/3334480.3382923#tab-contributors)

**Comment by PapadoAndreas** [comment]
[ΠΑΡΟΥΣΙΑΣΗ-An AI-Based Visual Aid With Integrated Reading Assistant.pptx](https://github.com/user-attachments/files/17479027/-An.AI-Based.Visual.Aid.With.Integrated.Reading.Assistant.pptx)

https://ieeexplore.ieee.org/document/9234074/authors

Muiz Ahmed Khan,   |      Pias Paul,   |     Mahmudur Rashid,   |    Mainul Hossain,   |   Md Atiqur Rahman Ahad

**Comment by Marigian** [comment]
[DarkPatternsFromTheEndUserPerspective-GiannakakouMarina.pptx](https://github.com/user-attachments/files/17513016/DarkPatternsFromTheEndUserPerspective-GiannakakouMarina.pptx)



[” I am Definitely Manipulated, Even When I am Aware of it. It’s Ridiculous!”-Dark Patterns from the End-User Perspective
K Bongard-Blanchy, A Rossi, S Rivas, S Doublet, V Koenig, G Lenzini
Proceedings of the 2021 ACM Designing Interactive Systems Conference, 763-776](https://scholar.google.com/scholar?oi=bibs&cluster=9533299911917692730&btnI=1&hl=en)

**Comment by apapost** [comment]
[Nikolas Martelaro, Victoria C Nneji, Wendy Ju και Pamela Hinds. «Tell me more designing.pdf](https://github.com/user-attachments/files/17644058/Nikolas.Martelaro.Victoria.C.Nneji.Wendy.Ju.Pamela.Hinds.Tell.me.more.designing.pdf)
[Uploading Αλληλεπίδραση Ανθρώπου -Ρομπότ.pptx…]()
